app/components/Pagination/style.scss

```diff
@@ -0,0 +1,33 @@
+.pagination {
+    padding: 0.5rem 0.75rem;
+    line-height: 1.5;
+    color: #0275d8;
+    text-decoration: none;
+
+    .pagination-word {
+        padding: 0.5rem 0.75rem;
+        line-height: 1.5;
+        color: #0275d8;
+        text-decoration: none;
+        background-color: #fff;
+        border: 1px solid #ddd;
+    }
+
+    .pagination-active-word {
+        padding: 0.5rem 0.75rem;
+        line-height: 1.5;
+        color: white;
+        text-decoration: none;
+        background-color: #660033;
+        border: 1px solid #ddd;
+    }
+
+    .disablePagiBtn {
+        padding: 0.5rem 0.75rem;
+        line-height: 1.5;
+        color: #0275d8;
+        text-decoration: none;
+        background-color: #fff;
+        border: 1px solid #ddd;
+    }
```

Comment:
padding, line-height, text-decor and border is repeating, use a common class  for styling instead of repeating it